.github/workflows/main.yml

```diff
@@ -43,8 +43,8 @@ jobs:
       - name: Composer Install
         run: docker-compose -f docker-compose.yml run test_runner composer install
 
-      - name: Test Stubs Map
-        run: docker-compose -f docker-compose.yml run test_runner vendor/bin/phpunit --testsuite Check_Stub_Map
+      - name: Test Stubs Map, Structure and Reflection
```

Comment:
Let's update the name of the build above. Currently it says "Run checks for StubsMap and cs-fixer". Probably it makes sense to change it to something like "Run code style and stubs structure tests"